MIGRATION-2.0.md

```diff
@@ -0,0 +1,43 @@
+# Migration guide from Barista 1.x to 2.0
+
+You just updated Barista dependency from 1.x to 2.0 and everything is in red?
+We did some breaking changes to improve our public API, and that's why we increased our major version.
+
+But fear not, here are a list of breaking changes and how to solve them:
+
+## Packages and classes
+We changed our Action classes to Interaction, to avoid confusion with Espresso ViewAction. ([#132](https://github.com/SchibstedSpain/Barista/issues/132))
+
+We moved most classes to different packages for better code organisation. You will find your missing imports in one of these public packages:
+```
+com.schibsted.spain.barista.assertion
+com.schibsted.spain.barista.interaction
+com.schibsted.spain.barista.rule
+```
+
+## Changed methods
+- `click()` methods have been renamed to `clickOn()`. ([#100](https://github.com/SchibstedSpain/Barista/issues/100))
+- `writeToEditText()` method has been renamed to `writeTo()`. ([#140](https://github.com/SchibstedSpain/Barista/issues/140))
+- `writeToAutoCompleteTextView()` method has been renamed to `writeToAutoComplete()`. ([#140](https://github.com/SchibstedSpain/Barista/issues/140))
+- `clickCheckBoxItem()` methods have been deleted. Use `clickOn()` instead. ([#140](https://github.com/SchibstedSpain/Barista/issues/140))
+- `ListView` and `RecyclerView` actions have been merged into `ListActions` with these common methods ([#146](https://github.com/SchibstedSpain/Barista/issues/146)):
+  - `clickListItem(listId, position)`
+  - `scrollListToPosition(listId, position)`
+  - `clickListItemChild(listId, position, childId)`
```

Comment:
Maybe move one line up the `clickListItemChild` so the two clickList stay together?